library/alloc/src/rc.rs

```diff
@@ -759,7 +759,7 @@ impl<T, A: Allocator> Rc<T, A> {
                 Rc::allocate_for_layout(
                     Layout::new::<T>(),
                     |layout| alloc.allocate_zeroed(layout),
```

Comment:
I _think_ this doesn't need to be a closure at all, I think, and can just be `alloc.allocate_zeroed`?

Reply:
Nah, I think it does, since it takes `&A` as a parameter too, as `&self`, making a different signature.

Reply:
Ah, I got confused between the impl fn call syntax and struct fields for a moment. Nevermind!